setup.cfg

```diff
@@ -38,17 +38,17 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    docutils>=0.15
+    docutils>=0.15,<0.17
     importlib_metadata
     ipython
     ipywidgets>=7.0.0,<8
     jupyter-cache~=0.4.1
     jupyter_sphinx~=0.3.2
-    myst-parser~=0.14.0
+    myst-parser~=0.15.1
     nbconvert>=5.6,<7
     nbformat~=5.0
     pyyaml
-    sphinx>=2,<4
+    sphinx>=2,<5
```

Comment:
here we should also drop sphinx v2 support, in myst-parser it is `sphinx>=3.1,<5` (see https://github.com/executablebooks/MyST-Parser/pull/400)

Reply:
thanks @chrisjsewell I will copy your pins from `myst-parser` 👍